--- extract_tmp/package/index.js	2021-12-03 15:50:38.727581225 -0800
+++ extract_tmp/package/index.js	2021-12-04 06:26:11.590492572 -0800
@@ -31,10 +31,13 @@
 };
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.DEFAULT_EXTENSIONS = exports.minifySync = exports.minify = exports.bundle = exports.transformFileSync = exports.transformFile = exports.transformSync = exports.transform = exports.printSync = exports.print = exports.parseFileSync = exports.parseFile = exports.parseSync = exports.parse = exports.Compiler = exports.plugins = exports.version = void 0;
+const path_1 = require("path");
 __exportStar(require("./types"), exports);
 const spack_1 = require("./spack");
 const helper_1 = require("@node-rs/helper");
-const bindings = helper_1.loadBinding(__dirname, "swc", "@swc/core");
+// Allow overrides to the location of the .node binding file
+const bindingsOverride = process.env["SWC_BINARY_PATH"];
+const bindings = !!bindingsOverride ? require(path_1.resolve(bindingsOverride)) : helper_1.loadBinding(__dirname, "swc", "@swc/core");
 /**
  * Version of the swc binding.
  */
