--- extract_tmp/package/index.js	2021-12-03 15:50:38.727581225 -0800
+++ extract_tmp/package/index.js	2021-12-04 06:26:11.590492572 -0800
@@ -34,7 +34,7 @@
 __exportStar(require("./types"), exports);
 const spack_1 = require("./spack");
 const helper_1 = require("@node-rs/helper");
-const bindings = helper_1.loadBinding(__dirname, "swc", "@swc/core");
+const bindings = helper_1.loadBinding("/home/alexeagle/.cache/bazel/_bazel_alexeagle/09a27d1eeea3b44d1580773090924b4d/external/swc_linux-x64-gnu", "swc", "@swc/core");
 /**
  * Version of the swc binding.
  */
