--- extract_tmp/package/index.js	2021-12-03 15:50:38.727581225 -0800
+++ extract_tmp/package/index.js	2021-12-04 06:26:11.590492572 -0800
@@ -34,7 +34,9 @@
 __exportStar(require("./types"), exports);
 const spack_1 = require("./spack");
 const helper_1 = require("@node-rs/helper");
-const bindings = helper_1.loadBinding(__dirname, "swc", "@swc/core");
+const path_1 = require('path');
+const binding_file = path_1.resolve(process.env["SWC_BINDING"]);
+const bindings = helper_1.loadBinding(path_1.dirname(binding_file), "swc", "@swc/core");
 /**
  * Version of the swc binding.
  */
